packages/prisma/migrations/20230420222915_add_whatsapp_workflow_actions/migration.sql

```diff
@@ -0,0 +1,10 @@
+-- AlterEnum
+-- This migration adds more than one value to an enum.
+-- With PostgreSQL versions 11 and earlier, this is not possible
+-- in a single migration. This can be worked around by creating
+-- multiple migrations, each migration adding only one value to
+-- the enum.
```

Comment:
@zomars are we supporting PostresSQL version 11 and earlier? If yes, we would need a separate migration file for each enum value added

Reply:
Should be fine to have them in one migration file 👍
Could you add all migrations into one single migration file?

Reply:
I'll do that.